css/index.css

```
@@ -1,3 +1,28 @@
+
+:root {
+  /* Colors */
+  --color-menu-link: #8794BA;
+  --color-location-hello: #FFFFFF;
```

Creo que con otro nombre serviria mejor, ya que creo que en otras partes esta este mismo color
```suggestion
  --color-white: #FFFFFF;
```